docs/contribute.rst

```diff
@@ -99,7 +99,16 @@ Adding new assets to rotki
 To add new assets for rotki you will have to edit `the SQL file <https://github.com/rotki/assets/tree/develop/updates>`__
 in the last update at the assets repository. SQL sentences for insertion differ depending on if we are adding an ethereum token
 or other types of assets. More information about each type of asset and columns is available at the
-`readme file <https://github.com/rotki/assets#adding-ethereum-tokens>`__.
+`readme file <https://github.com/rotki/assets#adding-evm-tokens>`__.
+
+rotki uses the following format to uniquely identify an EVM asset:
+
+::
+
+    eip155:<ChainID>/<TokenStandard>:<Address>
+
+So for example, if we wanted to refer to the Uniswap (UNI) ERC-20 token on the Ethereum mainnet chain,
+the correct identifier would be ``eip155:1/erc20:0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984``.
```

Comment:
you should explain what the format's name is add a link to the standard definition. This is not something we invented ourselves! It's CAIP-19: https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md

Reply:
Nice! I had no idea it was a standard :smile: